crypto/include/internal/ess_int.h

```diff
@@ -12,9 +12,18 @@
 ESS_SIGNING_CERT *ESS_SIGNING_CERT_get(PKCS7_SIGNER_INFO *si);
 int ESS_SIGNING_CERT_add(PKCS7_SIGNER_INFO *si, ESS_SIGNING_CERT *sc);
 
+ESS_SIGNING_CERT *ESS_SIGNING_CERT_new_init(X509 *signcert,
+                                            STACK_OF(X509) *certs,
+                                            int issuer_needed);
```

Comment:
_Just a note, not a change request: This is an example of conflicting naming conventions: OOH, the function is internal so the prefix should be lowercase, OTOH, all accessors for a FOO 'class' are normally called FOO_method_name(), with the classname in uppercase._